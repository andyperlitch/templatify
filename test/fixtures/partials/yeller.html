{{!
{
  "name" : "partial-helper",
  "description" : "A partial to test things out"
}
}}
<div class="yeller">
  This is a partial include, with yeller helper.

  {{yeller plugin}}

</div>
