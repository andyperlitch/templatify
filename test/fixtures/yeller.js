
var Handlebars = require('handlebars');

// exports so that we can require the helper function in our own code
var yeller = module.exports = function yeller(context, options) {
  // Assume it's a string for simplicity.
  context = context || '';
  return context.toUpperCase();
};

// and register as an handlebar helper.
Handlebars.registerHelper('yeller', yeller);

