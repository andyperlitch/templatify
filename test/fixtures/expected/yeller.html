
<div class="helper-test">
  TEMPLATIFY
</div>
